apply_diff.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+# usage `./apply_diff.sh 500`
+MODULE_COUNT=$1
+declare -A MAX_MEMORY=(
+  ["50"]="400m"
+  ["100"]="600m"
```

Comment:
Maybe for these two, we should do something slightly different.  400m is below the default memory settings for Gradle (512m). I think it's less interesting to try to fit into a smaller footprint than it is for testing that the larger builds fit into a "reasonable" footprint. 

Could we try these two as "default arguments" and remove the jvmargs from gradle.properties entirely?

Reply:
Done.